.circleci/config.yml

```diff
@@ -63,24 +97,29 @@ jobs:
             twine upload dist/*
 
 workflows:
-  # main:
-  #   jobs:s
-  #     - build-and-test
   build_and_deploy:
     jobs:
-      - build:
+      - lint
```

Comment:
Does it mean we only run lint when we deploy to pypi and not as part of the regular test flow?
If so, should we run it before?

Reply:
no, its required by the tests, and the tests happen on any branch.
actually, maybe I'll just have all three run in parallel.